src/errors.ml

```diff
@@ -59,6 +59,9 @@ let file_lines path ~start ~stop =
       aux [] 1
     )
 
+let pp_line padding_width pp (lnum, l) =
+  Format.fprintf pp "%*s: %s\n" padding_width lnum l
```

Comment:
do we want to use `|` instead of `:` here? Or is it okay to switch that as part of #1168 ?

Reply:
that can be done on the other PR, this keeps the diff small on this side.